tests/cases/base.php

```diff
@@ -871,8 +871,10 @@ protected function assertIdenticalInputProps($entity1, $entity2) {
             $this->assertIdentical($entity1->Type, $entity2->Type);
             $this->assertIdentical($entity1->Status, $entity2->Status);
             $this->assertIdentical($entity1->UserId, $entity2->UserId);
+        } elseif(is_a($entity1, '\MangoPay\PayInPaymentDetailsPaypal')) {
+            // TODO Add the right comparison
```

Comment:
I would call [`trigger_error`](http://php.net/manual/fr/function.trigger-error.php) as well to throw a warning:
```php
trigger_error('Missing comparison assertions for ' . get_class($entity1), E_USER_WARNING);
```